### ISSUE TITLE
proposal: remove 8.1.5 (regular backups) and 8.1.6 (backup security) as out of scope

### ISSUE THREAD

**Issue by elarlang** [OWASP/ASVS#1462]
Current requirements 8.1.5 and 8.1.6
| # | Description | L1 | L2 | L3 | CWE |
| :---: | :--- | :---: | :---: | :---: | :---: |
| **8.1.5** | Verify that regular backups of important data are performed and that test restoration of data is performed. | | | ✓ | 19 |
| **8.1.6** | Verify that backups are stored securely to prevent data from being stolen or corrupted. | | | ✓ | 19 |

Application does not produce backups, it's done by external services - from that perspective it's out of application scope.

And is it realistic to verify 8.1.6?

**Comment by tghosth** [comment]
I think it makes sense as a level 3 requirement, security requires availabiltiy

**Comment by jmanico** [comment]
I suggest dropping this. It’s more infrastructure.

**Comment by elarlang** [comment]
8.1.6 does not have any relation with application.

With 8.1.5 we can have discussion - is any part of application is involved in this process?

(I don't argue that backups are not important, those are just not part of application responsibility area - it's application security verification standard)

**Comment by jmanico** [comment]
I agree, Elar.

**Comment by tghosth** [comment]
Those are in because the availability of the application's data is critical to the security of the application. Arguably anti-automation is an infrastructure level concern or likely to at least be enforced at the infrastructure level but we still include it in the ASVS.

**Comment by danielcuthbert** [comment]
I'm with Elar here, I don't think this should be in here. Different teams deal with such things, and I don't feel this is appsec

**Comment by jmanico** [comment]
Agree with Daniel and Elar.

**Comment by tghosth** [comment]
Ok so to confirm, are we proposing to delete:
1) Just 8.1.6
2) Both 8.1.6 and 8.15?

Current requirements 8.1.5 and 8.1.6
| # | Description | L1 | L2 | L3 | CWE |
| :---: | :--- | :---: | :---: | :---: | :---: |
| **8.1.5** | Verify that regular backups of important data are performed and that test restoration of data is performed. | | | ✓ | 19 |
| **8.1.6** | Verify that backups are stored securely to prevent data from being stolen or corrupted. | | | ✓ | 19 |

**Comment by elarlang** [comment]
TLDR: both.

Application does not produce backups, does not restore from backups, does not send backups to external system - an application is not involved in any part or step in backup process.

If we don't put it somehow to clearly understandable application scope, I can not sell to myself, why we should keep those requirements in ASVS.

**Comment by tghosth** [comment]
Created #1488 and would request @jmanico or @danielcuthbert to review.
